src/d3a/d3a_core/export.py

```diff
@@ -135,6 +135,8 @@ def export(self, export_plots=True, power_flow=None):
             self.plot_ess_energy_trace(self.area, self.plot_dir)
             if ConstSettings.GeneralSettings.EXPORT_DEVICE_PLOTS:
                 self.plot_device_stats(self.area, [])
+            if ConstSettings.GeneralSettings.EXPORT_ENERGY_TRADE_PROFILE_HR:
+                self.plot_energy_trade_profile_hr(self.area, self.plot_dir)
             if ConstSettings.IAASettings.MARKET_TYPE == 3 and \
                     ConstSettings.GeneralSettings.SUPPLY_DEMAND_PLOTS:
```

Comment:
Since you are in the neighbourhood, how about renaming this to `EXPORT_SUPPLY_DEMAND_PLOTS` ?